internal/decryption/encrypted_audit_log.go

```diff
@@ -27,26 +27,26 @@ type AuditRecordType string
 type AuditLogLineKeyStoreIdentifier struct {
 	Provider *keyproviders.KeyStoreProvider `json:"provider,omitempty"`
 	// localKey
-	Filename *string `json:"filename,omitempty"`
+	Filename string `json:"filename,omitempty"`
```

Comment:
👀  made sure empty string is not valid or remove `omitempty`

Reply:
I can remove `omitempty` but we are not generating JSON only reading, so I don't think there will be an effect

Reply:
if this is for deserialization only we are good, I usually just wary of `omitempty` and strings/int/bools as it tends to come back and bite us when serializing